include/rangers-api/Application/Game/GameMode.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+namespace app::game {
+    class GameMode : public hh::fnd::Messenger {
+    public:
+        hh::game::GameManager* gameManager;
+        uint64_t unk2;
+        uint16_t unk3;
+        uint32_t unk4;
+        uint64_t fsm[9]; // tinyfsm
+        csl::ut::MoveArray<void*> unk5;
+        csl::ut::MoveArray<void*> unk6;
+        void* unk7; // created in the constructor, just not bothering right now
```

Comment:
Use the `INSERT_PADDING` macro instead of defining a `void*` and then add a comment. Second part of the comment is unnecessary and use properly formatted sentences for comments.